cli/docs/flags.go

```diff
@@ -219,6 +227,10 @@ var flagsMap = map[string]components.Flag{
 	),
 	RequirementsFile: components.NewStringFlag(RequirementsFile, "[Pip] Defines pip requirements file name. For example: 'requirements.txt'."),
 	CurationOutput:   components.NewStringFlag(OutputFormat, "Defines the output format of the command. Acceptable values are: table, json.", components.WithStrDefaultValue("table")),
+	Sca:              components.NewBoolFlag(Sca, "Set to true to request audit to only preform SCA sub scan."),
+	Iac:              components.NewBoolFlag(Iac, "Set to true to request audit to only preform IAC sub scan."),
+	Sast:             components.NewBoolFlag(Sast, "Set to true to request audit to only preform SAST sub scan."),
+	Secrets:          components.NewBoolFlag(Secrets, "Set to true to request audit to only preform Secrets sub scan."),
```

Comment:
it'll be `--secrets=true` , or `--secrets` will be enough?

Reply:
both are valid